DotNetifyLib.SignalR/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -65,8 +65,6 @@ public static IServiceCollection AddDotNetify(this IServiceCollection services)
          services.AddSingleton<IList<Tuple<Type, Func<IMiddlewarePipeline>>>>(p => new List<Tuple<Type, Func<IMiddlewarePipeline>>>());
          services.AddSingleton<IDictionary<Type, Func<IVMFilter>>>(p => new Dictionary<Type, Func<IVMFilter>>());
 
-         // Add web API support.
-         services.AddMvcCore().AddApplicationPart(typeof(DotNetifyWebApi).Assembly).AddControllersAsServices();
          services.AddTransient<WebApiVMControllerFactory>();
```

Comment:
This line should also be moved too.

Reply:
Moved from AddDotNetifyCore to AddDotNetify.